dsgrid_project/build_registry/build_registry.sh

```diff
@@ -0,0 +1,191 @@
+    -o ${QUERY_OUTPUT}
+
+# Create derived-dataset config files.
+runx spark-submit \
+    --master ${SPARK_CLUSTER} \
+    --conf spark.sql.shuffle.partitions=${NUM_PARTITIONS} \
+    ${DSGRID_CLI} \
+    query \
+    project \
+    create-derived-dataset-config \
+    ${QUERY_OUTPUT}/comstock_conus_2022_projected \
+    comstock-dd
+
+runx spark-submit \
+    --master ${SPARK_CLUSTER} \
+    --conf spark.sql.shuffle.partitions=${NUM_PARTITIONS} \
+    ${DSGRID_CLI} \
+    query \
+    project \
+    create-derived-dataset-config \
+    ${QUERY_OUTPUT}/resstock_conus_2022_projected \
+    resstock-dd
+
+runx spark-submit \
+    --master ${SPARK_CLUSTER} \
+    --conf spark.sql.shuffle.partitions=${NUM_PARTITIONS} \
+    ${DSGRID_CLI} \
+    query \
+    project \
+    create-derived-dataset-config \
+    ${QUERY_OUTPUT}/tempo_conus_2022_mapped \
+    tempo-dd
+
+# Register the derived datasets.
+runx spark-submit \
+    --master ${SPARK_CLUSTER} \
+    --conf spark.sql.shuffle.partitions=${NUM_PARTITIONS} \
+    ${DSGRID_CLI} \
+    registry \
+    datasets \
+    register \
+    comstock-dd/dataset.json5 \
+    ${QUERY_OUTPUT}/comstock_conus_2022_projected \
+    -l Register_comstock_conus_2022_projected
+
+runx spark-submit \
+    --master ${SPARK_CLUSTER} \
+    --conf spark.sql.shuffle.partitions=${NUM_PARTITIONS} \
+    ${DSGRID_CLI} \
+    registry \
+    datasets \
+    register \
+    resstock-dd/dataset.json5 \
+    ${QUERY_OUTPUT}/resstock_conus_2022_projected \
+    -l Register_resstock_conus_2022_projected
+
+runx spark-submit \
+    --master ${SPARK_CLUSTER} \
+    --conf spark.sql.shuffle.partitions=${NUM_PARTITIONS} \
+    ${DSGRID_CLI} \
+    registry \
+    datasets \
+    register \
+    tempo-dd/dataset.json5 \
+    ${QUERY_OUTPUT}/tempo_conus_2022_mapped \
+    -l Register_tempo_conus_2022_mapped
+
+# Submit the derived datasets to the project.
+runx spark-submit \
+    --master ${SPARK_CLUSTER} \
+    --conf spark.sql.shuffle.partitions=${NUM_PARTITIONS} \
+    ${DSGRID_CLI} \
+    registry \
+    projects \
+    submit-dataset \
+    -p dsgrid_conus_2022 \
+    -d comstock_conus_2022_projected \
+    -r comstock-dd/dimension_mapping_references.json5 \
+    -l Submit_comstock_conus_2022_projected
+
+runx spark-submit \
+    --master ${SPARK_CLUSTER} \
+    --conf spark.sql.shuffle.partitions=${NUM_PARTITIONS} \
+    ${DSGRID_CLI} \
+    registry \
+    projects \
+    submit-dataset \
+    -p dsgrid_conus_2022 \
+    -d resstock_conus_2022_projected \
+    -r resstock-dd/dimension_mapping_references.json5 \
+    -l Submit_resstock_conus_2022_projected
+
+runx spark-submit \
+    --master ${SPARK_CLUSTER} \
+    --conf spark.sql.shuffle.partitions=${NUM_PARTITIONS} \
+    ${DSGRID_CLI} \
+    registry \
+    projects \
+    submit-dataset \
+    -p dsgrid_conus_2022 \
+    -d tempo_conus_2022_mapped \
+    -r tempo-dd/dimension_mapping_references.json5 \
+    -l Submit_tempo_conus_2022_mapped
```

Comment:
This reminds me of lines of code that torc is replacing??? Am I remembering that correctly? What am I missing?